src/Debug/skeletonViewer.ts

```diff
@@ -5,31 +5,44 @@ import { Nullable } from "../types";
 import { Bone } from "../Bones/bone";
 import { Skeleton } from "../Bones/skeleton";
 import { AbstractMesh } from "../Meshes/abstractMesh";
+import { Mesh } from "../Meshes/mesh";
 import { LinesMesh } from "../Meshes/linesMesh";
 import { LinesBuilder } from "../Meshes/Builders/linesBuilder";
 import { UtilityLayerRenderer } from "../Rendering/utilityLayerRenderer";
+import { StandardMaterial } from '../Materials/standardMaterial';
+import { VertexBuffer } from '../Meshes/buffer';
+
+import { ISkeletonViewerOptions } from './ISkeletonViewer';
+import { Observer } from '../Misc/observable';
+
+import { SphereBuilder } from '../Meshes/Builders/sphereBuilder';
+import { ShapeBuilder } from '../Meshes/Builders/shapeBuilder';
 
 /**
-     * Class used to render a debug view of a given skeleton
-     * @see http://www.babylonjs-playground.com/#1BZJVJ#8
-     */
+ * Class used to render a debug view of a given skeleton
+ * @see http://www.babylonjs-playground.com/#1BZJVJ#8
+ */
 export class SkeletonViewer {
     /** Gets or sets the color used to render the skeleton */
     public color: Color3 = Color3.White();
 
-    private _scene: Scene;
+    /** Array of the points of the skeleton fo the line view. */
     private _debugLines = new Array<Array<Vector3>>();
+
+    /** The SkeletonViewers Mesh. */
     private _debugMesh: Nullable<LinesMesh>;
+
+    /** If SkeletonViewer is enabled. */
     private _isEnabled = false;
-    private _renderFunction: () => void;
-    private _utilityLayer: Nullable<UtilityLayerRenderer>;
 
-    /**
-     * Returns the mesh used to render the bones
-     */
-    public get debugMesh(): Nullable<LinesMesh> {

```

Comment:
This is a breaking change

Reply:
can you expand on that please?

Reply:
I moved the get debugMesh to under the constructor.

Reply:
Actually I realized they are at the end of the file can you please move the properties and the static at the beginning?

Reply:
I just moved the statics, move the getters and setters to the beginning as well?

Reply:
Check other files. Static first then properties then constructor then functions

Reply:
will fix that now then mark this resloved